regression/contracts/argument_sanity/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--apply-code-contracts
+^EXIT=1$
```

Comment:
This is the cause of the Windows failure: `goto-cl` exits with exit code 64 on errors (cf. other intentionally-failing tests, `git grep -w 64 regression/ansi-c`).